cmake/CMakeLists.txt

```diff
@@ -490,6 +492,44 @@ if (onnxruntime_USE_NGRAPH)
   list(APPEND onnxruntime_EXTERNAL_DEPENDENCIES ngraph)
 endif()
 
+if(onnxruntime_USE_OPENVINO)
+
```

Comment:
since we're not yet supporting windows. please add a check and message with FATAL_ERROR if user tries to build on windows.

Reply:
Will add this